sherpa/csrc/rnnt_beam_search.h

```diff
@@ -43,7 +44,7 @@ namespace sherpa {
  * decoding results for the corresponding input in encoder_out.
  */
 std::vector<std::vector<int32_t>> GreedySearch(
-    RnntModel &model,  // NOLINT
+    RnntConformerModel &model,  // NOLINT
```

Comment:
Can we use `RnntModel` here? Otherwise, I think there is no need to define a baseclass.